bucket/msiafterburner.json

```
@@ -5,6 +5,7 @@
     "version": "4.6.0",
     "url": "http://download.msi.com/uti_exe/vga/MSIAfterburnerSetup.zip#/MSIAfterburnerSetup460.7z",
     "hash": "d186cdf533727df267e0cb7cccfdb5673fbb8cb443ab009de023cae07345e31a",
+    "depends": "extras/vcredist2008",
```

Never depends on vcredists.

Users normally uninstall vcredist packages after installation (remove installatino files) so you would install them twice.

Use:

```suggestion
    "suggest": {
        "Visual C++ Redist 2008": "extras/vcredist2008"
    },
```